Description: The module Data.TypeLevel.Num.Aliases is generated using
 Template Haskell and contains a huge number of stub definitions with
 no real documentation. The resulting HTML file is nearly 10 megabytes
 in size.
 .
 By enabling the Haddock prune option we hide these useless
 documentation stubs.
 .
 This patch has been emailed to the upstream author, but no action has
 yet been taken.
Forwarded: In private email to upstream author.
Author: Erik de Castro Lopo <erikd@mega-nerd.com>
Last-Update: 2010-04-05

Index: haskell-type-level-0.2.4/src/Data/TypeLevel/Num/Aliases.hs
===================================================================
--- haskell-type-level-0.2.4.orig/src/Data/TypeLevel/Num/Aliases.hs
+++ haskell-type-level-0.2.4/src/Data/TypeLevel/Num/Aliases.hs
@@ -1,5 +1,6 @@
 {-# OPTIONS_GHC -fno-warn-missing-signatures #-}
 {-# LANGUAGE CPP, TemplateHaskell #-}
+{-# OPTIONS_HADDOCK prune #-}
 -----------------------------------------------------------------------------
 -- |
 -- Module      :  Data.TypeLevel.Num.Aliases
